src/sql/platform/theme/common/colorRegistry.ts

```diff
@@ -93,4 +93,4 @@ export const calloutDialogShadowColor = registerColor('calloutDialog.shadow', {
 export const DesignerPaneSeparator = registerColor('designer.paneSeparator', { light: '#DDDDDD', dark: '#8A8886', hcDark: contrastBorder, hcLight: contrastBorder }, nls.localize('designer.paneSeparator', 'The pane separator color.'));
 
 // Query Editor
-export const queryEditorNullBackground = registerColor('queryEditor.nullBackground', { light: '#FFFFE1', dark: '#4B0082', hcDark: '#000000', hcLight: '#FFFFE1' }, nls.localize('queryEditorNullBackground', 'The background color for null values in the query editor results grid.'));
+export const queryEditorNullBackground = registerColor('queryEditor.nullBackground', { light: '#FFFFE1', dark: '#4B0082', hcDark: '#000000', hcLight: '#FFFFE1' }, nls.localize('queryEditorNullBackground', 'The background color for null values in the query editor results grid. To disable the background color, you can use a transparent color like {0}.', '#ff000000'));
```

Comment:
the last 2 digits is for opacity, '00' means transparent. you probably should replace it with '#00000000' to avoid confusion.

Reply:
Heh, oops. That's what I get for just trying to go off my memory....